### PR TITLE
Completion of Barebones CSS for NavBar and Overview

### PR DESCRIPTION
**Pull request by BalajiSathiya**
Completion of BareBones CSS within Nav & Overview
Trello: https://trello.com/c/aBJ6i8dX

### CODE REVIEW

File: client/src/components/ProductDetail/Gallery.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React, { useState, useEffect } from 'react';
```
Comment (BTanaka11) [inline client/src/components/ProductDetail/Gallery.jsx:1]: the "React, " part isn't necessary unless you plan to use more than useState and useEffect, in which case for consistency maybe better to specify those in brackets just "import React from 'react' and call the functions with "React."

Comment (BalajiSathiya) [reply client/src/components/ProductDetail/Gallery.jsx:1]: I plan to test React.Context and others so it's ok

Comment (BalajiSathiya) [reply client/src/components/ProductDetail/Gallery.jsx:1]: if you can merge commit that would be great.

**Comment by BTanaka11** [comment]
looks good! ty for fixing "star.png" to "Star.png" for me

**Pull request merged**